Fix failing test

ddanielr opened the pull request:
Updated reference from using old row location to column qualifier for file path.

closes: #4774

ddanielr committed:
Fix failing test

Updated reference from using old row location to column qualifier for
file path.

ddanielr commented on test/src/main/java/org/apache/accumulo/test/ScanServerMetadataEntriesIT.java:
```diff
@@ -231,43 +231,42 @@ public void testGcRunScanServerReferences() throws Exception {
+
+      Set<String> metadataScanFileRefs = new HashSet<>();
+      metadataEntries.forEach(m -> {
+        String file = m.getKey().getColumnQualifier().toString();
```
This line was the fix for the test. The rest of these changes are just indent changes from closing the first scanner's try-with-resources section sooner

keith-turner replied:
Could this test do `new ScanServerRefTabletFile(m.getKey()).getFilePath()`?

ddanielr committed:
Use ScanServerRefTabletFile instead of direct CF

Use the ScanServerRefTabletFile instead of a direct reference to the
columnFamily.
